src/main/java/org/example/c22databases/Album.java

```diff
@@ -0,0 +1,49 @@
+package org.example.c22databases;
+
+import jakarta.persistence.*;
```

Comment:
Use maven to manage dependencies instead of IntellJ.
Add 
```
<dependency>
    <groupId>org.hibernate.orm</groupId>
    <artifactId>hibernate-core</artifactId>
    <version>6.5.2.Final</version>
</dependency>
``` 

to pom.xml

Reply:
done